jenkins/Jenkinsfile-blossom.premerge

```diff
@@ -268,38 +268,6 @@ pipeline {
                     }
                 } // end of Unit Test stage
 
-                stage('DB runtime 9.1') {
```

Comment:
please also include update to https://github.com/NVIDIA/spark-rapids/blob/branch-23.04/pom.xml#L747
otherwise nightly build may fail the revision check if no available 312db shim